src/main/pages/che-7/end-user-guide/con_workspaces-overview.adoc

```diff
@@ -15,13 +15,19 @@ summary:
 
 {prod} provides developer workspaces with everything needed to a code, build, test, run, and debug applications. To allow that, the developer workspaces provide four main components:
 
-Project source code:: the source code of a project
-Browser-based IDE:: a web-based IDE that is packaged by default in a workspace
-Tool dependencies:: dependencies needed by developers to work on a project
-Application runtime:: a replica of the environment where the application runs in production
+. The source code of a project.
+. A web-based IDE.
+. Tool dependencies, needed by developers to work on a project
+. Application runtime: a replica of the environment where the application runs in production
 
-{prod} workspaces are managed by Kubernetes pods. Everything running in a {prod-short} developer workspace is running inside containers and is accessible from the embedded browser-based IDE. Because developer workspaces are containerized, they are highly portable. And because they only need a browser to be accessed, they are easily shareable.
+Kubernetes pods are managing each component of a {prod-short} workspace. 
+In other words everything running in a {prod-short} workspace is running inside containers.
+It makes a {prod-short} workspace highly portable.
 
+The embedded browser-based IDE is the point of access for everything running in a {prod-short} workspace.
+It makes a {prod-short} workspace easily shareable.
+
+By default, a user can start only one workspace at a time.
```

Comment:
could we point to the config option / env var with instructions for configuration? I think it should be part of https://www.eclipse.org/che/docs/che-7/advanced-configuration-options/

Reply:
OK, will do.

Reply:
I see: https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties#L62

Is this assumption true: The reference file in the documentation https://github.com/eclipse/che-docs/blob/master/src/main/pages/che-7/installation-guide/ref_che-configmaps-fields-reference.adoc is reflecting options that we can find in following files:  https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties and https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties 

Both set of files overlap but are not exactly the same: Is there another file in che source code containing *all* available options, that we could take as a better source of truth?

Reply:
https://github.com/eclipse/che/blob/master/assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties#L62 is the property
Both property files are the source of truth - multiuser just adds additional set of properties

Reply:
Latest commit is adding the option in /advanced-configuration-options/. Should we really link to that page in the overview doc?